cpp/ql/src/semmle/code/cpp/exprs/Literal.qll

```diff
@@ -173,7 +173,7 @@ class ClassAggregateLiteral extends AggregateLiteral {
    */
   Expr getFieldExpr(Field field) {
     field = classType.getAField() and
-    result = getChild(field.getInitializationOrder())
+    aggregate_field_init(this, result, field)
```

Comment:
This should be `aggregate_field_init(underlyingElement(this), unresolveElement(result), unresolveElement(field))`